modules/nodes-pods-autoscaling-custom-trigger.adoc

```diff
@@ -0,0 +1,168 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: CONCEPT
+[id="nodes-pods-autoscaling-custom-trigger_{context}"]
+= Understanding the custom metrics autoscaler triggers
+
+Triggers, also known as scalers, provide the metrics that the Custom Metrics Autoscaler Operator uses to scale your pods. 
+
+[NOTE]
+====
+{product-title} curently supports only the Prometheus trigger, which can use the installed {product-title} monitoring or an external Prometheus server as the metrics source.  
```

Comment:
`{product-title} curently supports only the Prometheus trigger` -> this is rendered as  `OpenShift Container Platform curently supports only the Prometheus trigger`

But should probably be:
`The the custom metrics autoscaler curently supports only the Prometheus trigger`